router/rootfs/etc/confd/templates/nginx.conf

```diff
@@ -282,7 +288,11 @@ http {
             set_if_empty $arg_{{ getv "/deis/router/affinityArg" }} $prng;
             {{ end }}
 
-            proxy_pass                  http://{{ $app }};
+            {{ if eq $proxySsl "true" }}
```

Comment:
ah, this feels _so_ much better than the way the config is currently written. :+1: